app/build.gradle

```diff
@@ -61,12 +98,43 @@ dependencies {
     // Awala
     implementation 'tech.relaycorp:awaladroid:1.5.6'
 
+    // UI
+    implementation 'dev.chrisbanes.insetter:insetter:0.5.0'
+
     // Preferences
     implementation 'androidx.preference:preference-ktx:1.1.1'
     implementation 'com.github.tfcporciuncula.flow-preferences:flow-preferences:1.3.4'
 
-    implementation 'com.squareup.moshi:moshi:1.9.3'
-    implementation 'com.squareup.moshi:moshi-kotlin:1.9.3'
+    // Database
+    implementation 'androidx.room:room-runtime:2.2.6'
+    kapt 'androidx.room:room-compiler:2.2.6'
+    implementation 'androidx.room:room-ktx:2.2.6'
+
+    // Testing
+    testImplementation 'junit:junit:4.13.2'
+    testImplementation "org.junit.jupiter:junit-jupiter:$junitVersion"
```

Comment:
I was hoping we could use JUnit 5 in this project, but Robolectric does not support JUnit 5. So it's probably best to keep all tests to JUnit 4, to avoid confusion. I'll take JUnit 5 dependencies out.